src/components/metrics/_metrics.scss

```diff
@@ -0,0 +1,179 @@
+.metrics {
+  margin: $spacing-8 0;
+
+  &--margin {
+    margin-bottom: 0;
+    margin-top: $spacing-5;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  &__container {
+    background-color: var(--color-metric-background);
+    border: $border-style $border-width-base var(--color-metric-border);
+    border-radius: $border-radius-lg;
+    padding: 1.1rem;
```

Comment:
Why is this value (and the one in the media query below) using odd padding sizes? If it's related to the icon, use a negative margin to manipulate it as you need to.